From 1842eb22127481d06ac553794c1c245c626ec3a0 Mon Sep 17 00:00:00 2001
From: Yann Gautier <yann.gautier@foss.st.com>
Date: Wed, 10 Jan 2024 16:30:21 +0100
Subject: [PATCH 457/841] fix(stm32mp2-fdts): remove unused macro DDR_SIZE

The macro DDR_SIZE is no more used when generating FCONF device tree,
remove it for STM32MP257F-DK board.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git 1ca32076fd0c86ca1ed4eab57ac41c20479daabb fix(stm32mp2-fdts): remove unused macro DDR_SIZE]
Change-Id: If35853b6c85a5d2cb3ddd1a55dde45debc0e34e0
Signed-off-by: Yann Gautier <yann.gautier@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/350512
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 fdts/stm32mp257f-dk-fw-config.dts | 1 -
 1 file changed, 1 deletion(-)

diff --git a/fdts/stm32mp257f-dk-fw-config.dts b/fdts/stm32mp257f-dk-fw-config.dts
index 7f352b766..a99265186 100644
--- a/fdts/stm32mp257f-dk-fw-config.dts
+++ b/fdts/stm32mp257f-dk-fw-config.dts
@@ -3,6 +3,5 @@
  * Copyright (c) 2020-2023, STMicroelectronics - All Rights Reserved
  */
 
-#define DDR_SIZE	0x80000000 /* 2GB */
 #include "stm32mp25-fw-config.dtsi"
 #include "stm32mp257f-dk-ca35tdcid-fw-config.dtsi"
-- 
2.47.2

